Libraries/Template/Components/Header.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { View, Text, StyleSheet, ImageBackground } from 'react-native'
+
+const Header = () => (
+  <View style={styles.container}>
+    <ImageBackground
+      source={require('../logo.png')}
```

Comment:
1) What about using an inline base64 image? I'd be one file less to remember to delete.
2) Could you add an `accessibilityRole={"image"}`, which will be added into `ImageBackground`'s `Image`

Reply:
1. I think the code for base64 can be a bit scary for new users: 

`<ImageBackground
      accessibilityRole={'image'}
      source={{
        uri:
          'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYAAAD0eNT6AABdpklEQVR42u2dBZQdRdbHh2SR3Q93l0UWd3eHxVlgcYfF3d1Z3CE4wd0dQnDXDDPd/SYJgRDcIXiq+rv39ixEZpKZN3Wrq/r9f+f0GWCzmfeqq25dv01NAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADQmed6raXA+WVObnbmp1c7dVLPz0T/P35TYeZoG2lmahuSTy58BAIDoeCMfv12wrU9Cbd+mzJxCzxVNWX4H/XyAnifo6U/P402peYh+3kU/r2xK8zOaUnsY/f+2op/L08+ZIAhBlDTb6Zoyuzo9+9D+voCe+2mPv00/P21KjKGzkI/1kT9jPqGfr9FzO52NE+lMbEY/Z6czMR4WGAAQBomdgQTd9iSoLiOh9SY9v41TwHX5McNFCGZ5XxKAh5BSsFLTO/b/sOggGFI7CZ2BtegMHE/79UF6Pna3/zs8Ex/RczP9zh3JczANXgAAwC9ZPi9ZNCcXVo2msOvEOkrMAPrdl4hVlOVT44UAb7CHK7Gr0CV8Ou3DV+kZ4f0MjHwWUtNPlAEoxgAANYbkE5GVsysJvtdLE3idPYV79VQShMuRi7Q3XhZwCsfsE7u1uOMT811w+1+UgfxbOgfnSh4BAAA4oc1OKu7NxHwepOAb0yr6ggTh1eQdWEesNQDqgS3q1G7bnrfyaxR7vwgRUAjO9JHQHAAA1AVfnqk9QC7UaITfGMLwK1IGrpLcASRPgXHBSaecvJea6yX/JNp9354/w0m1UIIBAN2Cs/Az827cAnAMz8B7RTY1ZVIDMDIt+fS054+WPVKlPV/se8qXsUviJQMAxk6bnZCsn/MrJwRHFYiWvuPDJBQ3RL5AQ1v740nOSGpuJWX390rvef5+qT0UZbUAgI7h2v3UvFVtQThG8uBQcvke0dScT4EN0CAUWfxbt5eX5g223x+SnB4AAPiDzK5BwuGbhhOIf1pIP0pZIStBoKoX/9/ICt6P3vX7jbvPZa+/izAYAKAgtTtX3wXajfAAdyys5YtiY1SElnxi8fLEUsXiRwn4lJTdBbA5AGhsy38fCMNO3aUPSzIkiBMu4+OLPzNfYj93Ui5bs4tgowDQmJb//hCEXbKWHqOLZClsmEiQRFYpX/0Me7cLZbLc1RMA0FCX/7YQft0Wlg/AYgoYruio2R3oPX2Avdqtff2BDN4CADQAtXxNxPx7VEJ4PcWVZ8VGCghu3pOYd7BH697X72CWAADVv/z/3tjZ/s4E5i+kRJ0ls9tBeXDVBo/axZ504Qm4Bd0yAagqXAYFK0lh7oDdAw2FPMMjeFkBczp6Go/kTgAAqugmNZdDyClOI6zZlbHJlGELtWa3or38EfadihfgV1ICFsJGA6BSrn+7HoSbJzcqprDpkNh5SNHqhz2mvoebZfQ3AKACcJvbzHwCweYtLPAdXVb7IizgCC7r43HUUY3kjd6jdTI2HgDVcP33CWY8aWaG0DOQnoyETCv9bGtK8m8rakm9SS1Xl8AG7MnetSuQQpVURDEcRnviaakiSfMzZFRvaveUTpzSjdPuRX/uSPozF9Cfva84J6Xt3d/QHwCA+F3/S7e3ty1D2F1LQm17+gzLNA2y0441w5hdjlxax5+X/z+ZOZuex6P3XCTGyHfhBEzQdaR9r7molL3r5gKleQPm5qLZFikxnLRY1zrYKcmb9C/6O/vS3/eD5+/wNKoCAIgVPryZednjZfcT/b6LSWAt6FRw8Kz2zG5Jf/cV9DsGRaoIDJJadTBuUrta4SmK6sL/np676B3vqjZoh5WiYpjRx/6+l90YGxKAOAXpZt4yhzNzGsVqp/HyvVrtbCJopf47trgwKTH1WoNVhxvR8ETGeN7ll4VSatemy3kCr96R1JznxztCobqn879gcwIQEzzzPDM1L+VvZcYKuREPt3/lOefR1ISTdZvYVbBJRyKxy0o+SAwNoFJzm1TV8BkrV8Ffp72Xv+53rtndsUEBiAmJo6sLw9ubhtq/BvOdpdqBPAOJeTUSReCChs8N4EuUM84lVyLoi38A7a29g+v8yMp3aoYqf/f34AUAIBbyvJd65nRqrgq6zC2xi4t7VioPgr5YEmrPvGhD7tMWO1fQylpiRtA+v1VGQoecDMftvdXzAsigAADEYP3nmygLxzuiqXFvs5NKeZWUHAbrCfiNLplDRXFrHA/Vdv6z2rvRx4HbDA+0s0Sznty9T1PZZUUVFQEARKEAPKNa2x6S2787XpHUbkSKwCsBW5xPkcIyc7WtfklguyHQ9f9c6vAH55NFubY1u6luLgBNEQUABEyrXUC31MnOEfX6sBVTlJk9EWxmOSeYVZHCSs2CvPhTe0gl8jFUqyio6gYA0KACIM13qtiFtCSt172BKgJnl55l7lLpKpIzfw5sjb+mz3WElB9WBfbOaXUQ5LJDLsEFAAQId9PTaqvL7umqxgC5BC01zwbYj/2V6AUuW9XSxS4oi58UEXN6cBn9rsjsGorJgMdB0AIQpkW7mZLANBSbnr/Sa1eMmaWJiebdAK3Uf0a5pkWW/4DAFKu+lc+zECXA3K0kCwYjGRCAIC1ZHiCiYole2TBryNUN3FgoMx8EZLFaCr+cGFWVQGI3CGrIU2JekJBPo9Bq56Y9/LvSeq4IYQtASHDmskonPPo7eUhPo8HhlNQeU8w3COYSe5Qu1qkCV6B6FaN7g/GgfEzvcZuGtFplGJfKml4IgQtAWBbX1krW/9UNva482CWoREFqI1yzi4S5VjTjIDX3BNTE57yGnrtQhGCMwtoOa6ieFQCEr+1Tcx4NQVr12H9Xqdn1Jf4ZhhLwI1nZ/w5qfdrsnPS5WgK5/F8NVknyrpQpKa+cOAsACEL4TqjSVS01/bC4I8FhAXZvB1PORpnsIXRkTPNVvQylGfd6DCeLf/9oulR6ezcqa30aFheASh9yuyUWtyNvgPRe7x9IqeDD0u64NM+T3U0x2aw7F9JjqFHvAOnBoDAVNDVvYXEBCIHMnKpwwL8Rixd0Jlh7yYyBIIYNUeki5yr4/f696feeE0Tf/tTugtK0sSppR6isfUs+PRYXgLLRmKjW6Ml/Xfe+zC5NksJoZ7u8l+/MnfOCSIykds4xDewpC+57oDIbgMplAQBlXkCU5ayR6ZvadbC4sXkDzK8yZU+TZjsd7bfXSlZ2qDTT7gOrv1tGwgswEgCoGjyhS8OtWpU+9I3oDeB2rRqXIzeXScx7JSs5b9LPebHZuklmD1R4FwOxsACUe7BPUDjYd2Jhe+AN4MlyKk2ZuvUOr3WqxNXs0nT5f1HydzqL4s4TYJPVgZRpaiSh2hmxuACUpgCYxxUO9S5Y2J56A6jtbGIGlR4jd1EhwD0QpPdAaS7/z6jufC1sqh7LikxBVmyGhQWgHGtzPJkf7/pQo5zKldU1Kb2fm0qOlw/okZVWjPEdUWKZYz9kmztTAC5SGBF+BhYWgDLgDGj3F8Z7WFjHFMOFSkwQlMFG3Yubi3JJuQTlD0BCUx9nCkC+icqYcABACaR2QwWL6wYsrAKJnUeap5SnBHzZ5fatRY1/n1JLGjm5FbilOZ9CFCvXCcOYCwBAGRq9hoVG5WxAB27ZnJjLSp0hMK7yTk4czMwtJX7Gl6VuHSjJDIV5Db6bUAEA5DDfonCYl8DCqntudiZF4JeSLtjfOh0kJLMOzAMlxvsvRZa/9t6j2n33DYHWx8IC4P8wO3YpU093tlKBDyVgSXp/Q8uLr9s9Rvk8LfnE5fUwoAZGrBQBfWp2dwWv4eFYWAB8It3nXJdmmWYsrEfa7DSlNg7ifgUMx4bZ9V7O5f8xXUrLYDN4VDzde26ux8IC4FeTn0lBGN+ChfXM0/lfaN3PLjHmfnZT64gBJXkiXkUjGc8MtX91X9ZpXsLCAuCTzK6g4Mo7Hgtb2vvcstRmO/4Vj5swbbIkEpM4b9QEAPBIardVUAC2xMKWqgQsTML0w8pf/qk9FoN8ypQdCpMceSgZAMCbAnCMe5esXRwLW/p7nbF94E31Ln6pfICSWf4eo+59zisB8kWxsAD4QqOevMVOiYUNgHfs/9H7va9il/8XpNwsj5cbhJK5i4LxsAEWFgBvh9i1G8/8ALdsQHA3vtScW5HLfxApl3PhpQajAKymEALYAwsLgC9cl22lphWLGuJ7ps6MZQ7j6bli+XrTIDstXmRQe2oOhfd8ChYWAH8KwBDnU9dAqAJ7XXrf30do+T8qDYZAWHC758QYxwrAtVhYAPwpAN87L8sCISsBsVUI9JWLBoQqPz51LD8exKIC4EuDd9/N61wsbODwkByNYS7u3cGnIZ8kcFLztvMhTgAALxfBNApJPEdjYSOgGOn6fKAufxo1a/fBS4oADs+4VQAGYlEB8HJ4abY8xgA3LtzONTP3B3b5j6B9uTVeTjQegOvdvv/8WywqAH4UgMUV6nghvGOCZwi4FuI9afCT2o3wUiIiMxc49/7wgDIAgDI1u5LCTO/1sLCRUUyEvKDkeP9w8h6tgZcRmwJAcz+cNxJDxQcAPg7vugoegFWwsFEqAeORJ+Dkcnr6m29o3yyHlxAhqd1PQQGYHgsLgP7h3UwhB2ApLGzUngDP8wMse43Q/S1eGbKLQitxdHsEQJ2a3UrBA7AgFjZaD8C5pbn/a3YZvIQIkfHTGAgEQIyHd3vnh7fVzo2FjXEv5EeWnAPwJf2cFy8iOg/AhvAiAhDn4d1Z4fDOgYWNThHcLZDGPx+QJ2AmvJCIqOVrKsiQFbCwAKgfXru788M70M6ChY1qD2zqvp97j5SAd6VJEYjEiMhXVWgmthoWFgD9w/sfhTJAWHAxCW+uvQ+tE2Bqnmsakk+EFxSF92gFBQVgHSwsAPqH173rl/vMgxgs/0XCngxo7qTExN54UcHvo2UUQgDrYmEB0FcAdlVIApwNCxs4rKQlZlgEw4AuxDCg4BWApaEAABAjSAJsPAbnk9HF2hzNOODUHoKX1mAeAIQAAPDiAdheoQ/APFjYQGnJJ6D4er9oLv8/L4TN8fIChTs4upcha2NhAVBXAPItFDwAC2NhA6Ro9HN9dJf/n0OClsdLDNKIWEFBAVgLCwuANhpNPDgmCAJ81+akKC//P5WAL9AiNkgZspqCDFkZCwuANqxp4/A2gpW2a9SX/59JgTX0CAhub62rIEPQFhoA/cObr4hxwA2g5CVmRDUUAFEC+ksuAwhFhmyiUEq8GBYWAG146Ib7HIAtsbChvF87H13+33mb6udPCbgG5YHB7LGtFBSA+bGwAGjTZudUuAh2w8IGYZlNTZf/YI9jfQ+iPINnPf6+w/GSQ9hnGuEllBIDoM8gOy3qtitIUe73rMfWvSfJ7x2ST05KxwBPSYGW9tpGeNmlKwAHKVQBTIWFBUAb7rfu3j17Cha2RNg1nplrPbrj+4zijk/tjPTfhnj63T/Q71sIL71UBeAE5+8VOR4AeCIxPzu2zC7DopZIag/zmJR3R4f9+rkZFJft+VEC3hdPFihpv5lzHb/PX7GoAHjT4M1HjhWA27GopV3+G4lr3I/b/0nKIZmw088iPeLNj96mB8JqLEt+XONYAfgSiwqAvwPc7FgY98OilkAx3W+4pwv3LVI2Jhm3d8luQAqJ8eQJuByboBT5cb/j99iGRQXA2wHOn3F8gJuxqN7f4dTiCvdz0X5AF/sM3VBMdvdYGbAXNoNnEvOC4/31MhYVAH8a/F2OQwCfY1E98nT+F2mO4yXzPv+2rhptTgz1o5z8TsrJKtgUXhWA1LF36WEsKgC+SM0lzsuz3sjHx8J6e3/ne7pcf2tK81Xr+oxFZcLNnsoDP6d8gFmxMbwpAF84VgBuwKIC4O0CsccqtAOeCQvrgZrdweNI3m169Fk5YdBfb4I3mobav2KDKMOJl+7zS87HwgLg7xLZXUEBwERAfcVtSRmT68Wqtke5uTDslOQJyDwpAX3RLliZgXYWhTyOI7CwAPi7SDZUsBY3w8Iq0myno8v/Q0+u/yucXqQ80jczX3lSXPbFZlE1HpZWeGc7YmEB8AVP3tLoCw90YLdrYp73dPk/LkmG7i+OlSWnwEdSYGZXwKZRIrMbKygAa2NhAfB2iLmEzHkc7zwsrJbHxlzqKZkuaRqcT6ZoPXrKXzAfk9I0PTaOigKwj4LxsDAWFgBfsHvXdTvg1NyLhVURuNt7ujS/lEmR+srMf711CkRlisb7O1NhFPA0WFgAvF4s1H3LrfX4DhbVtbCloTeJ+clLuR+76P0on73oO93nSQk4F5vIudy4xbHc+AWJmwD41+SfdHyJfI+D7BB2xbtW0jpP4NzZs2IzifN21EhO9YP7LoBoAwxACQrAlQoCd2osrBMreTx6P/d4cv2fVc7+y2f3Mz2QFFOeVAhcKQDDHHsAnsKiAuD9IFOdt/ts3mWxsE4sZE/jfWmoS0ejff3twVWKrH3179lM+QB/w8bqIdxoyf27uQ4LC4BvMrsl6nlDvPyp9a6XaXrmXcqUnziAfbiXJ2XnGmyuHsIzIdznaZyEhQXAv+BdSkHIno6F7ZHlPyOt4aceLsOvvWT8d4ViZsAVnpoEQUHt2f7cEO8EgCrQnE+hcLHcjYWtE26+46PZD3sXQmu8Iv3lzYsevvtPTa12AWy2uo2GgxR6AKBpEwClwFPU3LrzWrGo9QpX8p74yYo/NMy9aGeQBj4+mh29Y/8PG66uPdrH+ftAwyYASlMAnnfehpWtOdA9UruOpzj4LUGXaiZ2OU/tgq/FpqtLAXja8XsYjtJhAMo70NcoxPQWxMJ26/Kf0bknpmPvzNtRZMKndg8/yhB1WATdlRefOK/OAACUJmwVys2ougB0DS7Bc25Vdej2/oJi37NFsibjiYWu7wUYTvv/H9iEXWRIPrnCe7gDCwtAWdTs+gqC9VQsbFcVsPxEL0l/mV09qnWRenPzpoe1GUAX20TYiF2Ae3y490qdjIUFoLwLaHYFBeBBLGwX4Es5MdaDq/vwePem+cqDJ+AibMYu7dfdFBJSt8HCAlAWPJiFXaFuraoPsbDjoNlO5z6e2smExpiTrDL7T0+VERtiU47rXZgLFaYALoaFBaBMEvMaZgJ4V7oe92DZDpSBQvFfPKd4GYVcszNhc471PbjPVUE5JgAlk5qrnR/sWr4mFraz9baHeGl4k9mFK6Iw9aY92s+DEtC/1LkIYb+D8URJcrtH38PCAlC6B8DuqyBQj8TCdgC7PL3UuVesxG2QnZbW7SMP63YENmmHSuuMCkrqfVhYAMomy1dES2APcA1+YlIPlmyfSq5fza6sPySJGlmldkls1jHWXqNa6BQsLADlW6WTKiSfDcXCjq5omcs9uP5fk3rtxE7VNNDOInXu4nUgJY/DMondgP55C/EQ1OzuhffHHiwem8weJ0I5zc+gn+fQOzxfMuQTc5kM60nNVUXjKK7RpxGuWd73z4f/Xf77NRJSSs2V7f+/i9v/nnPk7+WyL/k9ZGlLX3m7T5Fdbrejz7q5fD7+nNwfnj83f37+Hvx9uFwvtUd7UKDaEJsefe/yO3O+1ltgYQEIgcQMUugIOAMW9g8LalMPl79tah1h/HTRK+lpHTGClIjfPVRPXIVNOxJcTeJ+nefFwgIQhIZPHbncl1Zt1PDrylYrN1BJzHeVvpir+KR2Z5nQCNh79YHzJFWsLQChaPg0IQ4xvq7D2eJcNsaXe2o3o2d/spLOJMF2o2STc6w/Nd/gIo3e42CLOQ3Usz4xj7aHOE6m972n9A6o5YtKeKLKA224pNe9bHgZQheAUEjsKgqH/PFo14MT9mp2Pop9rlsIe4lf31rMrGdryIMbGk88j5RdUu6AlCtynoQ9ns7UjnKueAZDzOWFNbueQojlUghdAEKhJZ/YeYY1W8ChWkb8uaS0jJLNWFCn5qR26/1FLx368DTYQwqj5NmYJyShkpMgE/sv6dUQ+oRGPhvuwyu7QOgCEJSrz7zrviEQWdHlhjYmIet9Cek5zoJMrPj8DcTk8QSmIHCPg2eKagtqFMXVEImdJ4g4eWYeU2gUtigELgBhKQDuR7Cm+X88hTCmImVjpWKWPPUsT82TfhrH4MGjqhhQ0yjTQs+dosDW7Fa0xxcij90EnjxlvdznsphfvX1+AECXreU9FWKjNzr9jDIm1i4lLkSuLy8uerjs8TReSCE1rXS+bm/vq7CxTE90HXJrtQsoxP9fgbAFIDQSu7iCoHq/7s9TxOjXlSY1qblNMuvVO8HhwRNzMmL+LZ2V56SBE5cx1uwilGMwfv1GAXnw3MuECyBsAQgNFhSSzez4wHMnt3HRkk8vZVVpfiIJiAfocwyDQMeDx5HLnfNeOL+Auz9y/L2rSkFqrldoELY1hC0AIaIx8pMT8Ea97Cemi37Vov0szQzgtsEQ1Hjw+Cxb/Flq8Tlfhi/kWv73DsMHiRmskAD4dwhaAILMA+Be7c4tkFvowt9JLJDEvAM3Ph48QSoFn4v3jRVzTqgdaOdSkAWfVLppEgCR5wGsBWGIBw+epoTmLmBKKACN5AGguvnEjIAAxIMHj/uHJk8CAELOA+CEIQgrPHjwOE8AXBYCFoCw8wDOh7DCgwePY/f/jz0qSQQAeKDFbgthhQcPHscNgJ6EcAUgNDgrN8vnLUYCUxlgbQQm3eHBg8e1AvCZjMyu2ZWDmHEAQMPCrrjMrlHUASvU+uLBgwfP2CaFcnlwZrdsarOTQiADoB7bpwz/LN+CDt7N7gd84KlMj/nMDG8X0F+214Z/IoOVEvMh/fyAfr7XPtq2jZ5a+8P/PFD+uyiUZkj7n/+Yfn5GP7+SCYzcgAZVJnjGHHj0uMwi4Y6gAABHDMknJ5fbDnTA7ifB+wuETfTNWmz75TyEfr5FP/vTz3vpuYH+uQ89Z0s75dQeRtbVXvLuee58ateh//+KMho5sQvSkJe5SdjO2tRsp5M9wkOWePKbv7BTL/q9E4n1xzMf+LPw+FuedFezy0inyJpdj/5982L/0nfh78SNqlJzHj1Xtc+HeJS+80syMa9QOH7APol+f9MsA3sg7Y2ZIcAB6C4sVDO7PR2khwvtGoIlqkzpxJxK7283uvg2pUtxFRno0mpnaxqcT+b1ko45vNVmpxGFgpWJzP6Tnu3kUklHfIA9FpVC8AIpfvvRu5wBGxuAzpAxueTeT809sPSjrpPeAJtZEe5DzyEO7LX4PAOcoMzTCVvslNjIAOR5bxJoa9LBuE7iqxAUsSdGXYVN7QEOJ2C/xZ0zkJj7JDzEYSQAGsuKsfPRIThLEqwgEFxbGr/Qk9DaPihzy9n9WLPr0+V8pbJQ+wDZ0N4U5/GkJl3zfbZyOS0r5xTKyczp9PtupX31miRD4py5ria4XDoMYsgQqCwc/03tHrThX8HBdzEf3bxLz10inDO7qySdDbSzdBhj5xi8ZMeruv7Xxib3SJrPrp4w2Nnce3Zh1+zSdJ6p4ZY9oSiH4/bbSGDsofKe0JoeIkmlAFTCUknscu0u/p9wyOsYb8oZxWK90xASziov5p737laYRSw3uP4RCuj2/vusW/Hq4rzPQJ9rdXr2oX1xCf0dTxVlmDjP3SxlvVMmmyJpFkQHl2Uldl/axM04zF068MPFM8IXPbvs2ZrP8qkdXRIHKl8SH4p3B5ShYPeSMsoYlLvETiUd9DK7t5R7cnZ8Zr7H2R/n+RpUlMQ6kgcAqNFqF2iv5UaW8tgONGv3mT2O/n2Tdou+l9L7mE3/XVCJGiiPoirgR9V3XLMrqSkw8vnpHHDvB+71kZqhkBOd5PawJ5X7YAAQkBXSWw6wtiUSp2XfJslTPKMgtauJZ8Rr+MU8ovz9rsMBCIDUHqAem26zE/oLbZC1y+7vLD+yPc/lfciS0XoLZPbfmEcAyuMd+3/i5heLFodSLJfE3C5JPOzCL9stzgJC9/L/qKk5nwIHIRQlnLsJau5ve2yp35EbI7G3iT0FRffEryF3qPKG5Q1CcMAb3OtaMs9xAOnS79ve6nWmoN6RdFJULrFEw5/AQgFSWvurqgu6xc4VkNIznnRKTOyO9L2vaEpGfNvAHoHvpIV2aHIIVEnASKyxD7r0BWQVdWr9U/2/ruVxEw5EkKGAY5Tf++NB1qq35BOgkVh7g6HUXC1zMgBwZE3OLwIfE9A6OnBvBvi+FqN3ZRS/85fijgXhISOytStvKLQUnuKzDmTRKB4BUzRuoqFZANR98RfdvywO1dg6plGmfThu0V76jZZoSBMIORSwtO6ZpZr+0GLOHAKALOrsfVESpV0YBwN0VZv+By7+boUBDgjo3e2hLEyeQLvSGM6wOV95H1wQkNLbWxoWQRaN67lD8kQA6MTin1niR6ru41IT9lolc9393/1MEO+P24dKX3E1t+JPkgcCwqcln1i1dI5lRC1fNIjvyuOmNb5jy4hfKhkayPK+0kYagEJYUKtPziCtXHIfJcSwxcqd9TI7R7uFfKjKoQohJp6a65U9HYfisEQVClhP+Xy9FESb2sxcrFLS+1H+N5Ib6xZ/f9X6EIhsvBC5PI1tJUxAG/ygapXzUYIaX4RcmtfRZDouY9KJi+9WSSto5GRHNB2JD+5LoasU7lK6+1+j3DU1l472e8aTODpX/WTm9UqVDyb2qKah9q84LI0Cb+aa3ZQ28sCKbOJh7WVvK3ZpSI5M0XMuMJ4s7X0Wmd8tius7goTE4jg4MSoANJgnyb9V3BtflNoMSmYLqCj0a4z19/LUTfEs0sjkSoRMqaEQT35Efk/FKTL7+1Wi615qzpNJg911Q6bmZJUwQFkjPLXbwHJ4CMRLavdUPouXVMr9n5mvuuXtYjd6mv9H5GrsygC3GMasgUpe/JPSBj1XfSa8tqVfXPrL9ij2yHXyOlbDXt7fKysdmhYeWwbc8hnE7PHrpdomuKyEwML9rzBm2Fzbo/PIcoBHeMcrZ6n6y1wu0xtBBdz9qd1MvS2s7pjc68Ul1xX3flfXRCep5xn/1h1VbejW/G+MQ1QBanYR1UZebDn6dh+zTAi5xXVLPit9xiNUw3Pa+VSZ3TWIRE9Q7wY0D0SpgbI7rWZ3kHImHdfhBTqf287oUQAupdzw5QEcogqRmXMq1SBKQ/nlpDjXUw9ZMRKvI8scvlSjC7c+h/4B0bn87N76c+CdXziZZKSy4qJvEa0UdVMg7Y5/XPOPWuGqGQQT03v9UPH8fkz7fxJP32UClZ4XqblB/3OTRzY1D8eVL8BDpuxx8vlBwLDQzkz/yOpRb2nP4PfnQuQLVKoHFGqj/bznnZQn/R2Fw1RBEvsvXWsxP8PP/rcbKSnwG3p7F1xJkNkTdJUy54bBAFQEhWn1c2nf7nQB/RDJxf++XDLNdrrylCWldqnaswG4D7tm69PEJND0KywnEvOIqqXoYxKdRn8DDfd/V+CKA1ZoEvNoJErACKmk4vJjEABchpKY+6KI7bPw4SQbVwl9PbOGllX6rkcqKy7nKltxq+JQVVpezKnb9dM8qPz5J6XP/7OC+//60t8NK0+cKxDFaGNqDobcgNIvsbV1SmGcZ/KfH1wfea1qALagtcIZWT6vailnYm7EoWoA0vxE5YTAdRU/+07RfebuhzgmIdm+b/DN2kQRo7JHNBAqwW2UmVODntjHburUHiOzBkIlM2cpWdE6zTQ03bfcT6DMkAzwB7d+Tcxg1TCSlotYpZEZZeeH6NKWXgf5JrSezwdeKXBv0HK+Ylb/DNKCMlyLv006Yw3JJ4pgLRePZlxqZv+p3Nd9fxyuhpIjGyjvp/2cf2ZOnFMxekyfCN7XsmGHeqWd8LI4WLqbYLlgXf6JeU2yjEOI73cvDDBQxfvh0qLgvysxqeLhbcGwnwaE4/V6e+pr593kUnu0UtXLKtG8s1a7gITqgiwjpPAkGxIICWgcVpo4J2Vzwbl/3mpP7BsvUiF4qsq61Oz6DgXffsrW2mo4YA2IfkLgxY6V9ZrKXJEYu93JZFNznWqHx/rf+81ktPwNB8wFRbz/4gBf8rvtFn/c2h5r1DqK0W1OPh9PW+MBJXrv8o6GOk8s7DleyZcfD8fiNrm8B1igckVNTB4sJ8qlwnCskUvGXGWKa1Xt+OpdoKnEZXnf4DwCbBj6aOpWabh7V2oeCq5jX81uVake0bxZNTJkuWa/55/tXEUB/VNlDynXdKd2efGepOYqSaTipivjEpTFMJSP2wfo9CXP28FSGlnVoUhsqbEVrHcRPOTIU3eFkvdroUq8R1a0MnNncIngyAuoW+OdQeViqv/i/0SGQ1QxVpzZg5RKi3btuXavGfah9p5VQoSgPbx9TvsvzuObmXm56N5GyaNVinNm9t/KnSXX6rGSolIbb5qr59GxSwbVDZbPIbc/Bt2x/MkdmZj3gnmBmTnNW5/vcjwt0+u40HrYGjgzdykqdEOkHCx2uHSxuPSbPSvEA0lBP0m986OfsMh4Ms1Sb62aexRaSe22Stb/oZWUZ0Vn2PVof7aGM+BNoSqkkvBs7cx8GsiLu71hhsJoteKsNwaa2RWU3+8mFbB0blFtjNT16ZUPSSJlzF6BzC6sGkfm0uD6z+ZTCu/NiJe1ykj+GDXqCWYKISVco0JgrC7MpaUhS/kv6w0Z0NNIpHYbpU1/Tn0WGbubKzS/3eUZyczjgTZEeUXc3bGubWIuUxT+n9blReQOojpn4NGGkW1FIvHFYSQKmosqlT/mULAtU34PaG7bSyNtGy0TmuE4o8YwpcR80e3hOqndXDekY+eI7v2wq11jCIzOOXpcKgyi8wLkU6uM2f1TQTqpjjCYVpnuVg3qXX4pAEX5KigBo1/+mfm+5JfycCXimT0TgH2VYo1dT4JhZUG1B7g5PTo3ZmoPkYqFaMZct5fA8VrHlmeR2QNVq05qdqbuubDNRyptr6uQ/1IPfPFyOEZT0euaHLoGSkBh7S0kXbPKLNUoyvoQm8nsGkpr/Eg39oNi0x+q5IgpmZOTYdmtHtPF31HZbEwz1LnrJH9mTcHf5ZCEVrtic3nDy7qipfxdJZ+NPo197xRlXiW29jU3YYjDKNpxb5WaaI69cR/zce+HSSVkoNfxb+eIBNTWEpKK+vL/45z9JpZ1LMKOM8g1kyY54bBr7v8HlNz/S0PY/WFwbCZGYHme5zMb1d08dWnjHtkFxgIWdHAgtDqjdaHmXrMrG/eUiMHlVihh51fj4h/jHdwajes5M48pKgHj9ogVg3+MwjtohbdzDMNjmnK9AdRsq8EWfMLSRjzy7230WP/YwwBzKFmB7481uZJ7EWTmx4bu98/d9rSsvnC8AS83DbLTBv8uiqEzilnjdvWxK8P5iaj996p4c++AHcrLRaNmVA2z0DzNqYykpNQe05AZ/t33AvRTcj2uN5bfeanipfNA8GvObZMz82K1L/+R8gLa7MwReAH6qJYad+aR4uS/xAxTmVYXg/JVrnHKA6JeLaXhHE+7bYCYywElCJxPG66uvydI/FnlPTzYsfUvk71+V1P8snzeoNd7SD45ff/XG+PyH6kTY1fyQsp2DWuWJndWipfZjZXW/C4Ity7AlUiaM0jGdk/FoBjXTc2u7L1zWWJeq/aiqlxIE6lUZnACVEfhF44N68WdLwlc2EzcOJb/GAKvTVoZh22wHKYom97rsEeGWqMnuy6EW7fuq02996Zh7wOHyCtHkfT3sWch01cuM1CP+1NpBDPNVRj1kC2ieMF8H7TLU2r8Q5t2WULXzZAnDRbK8BDFQUH7juZ9m0dJER6K8GddCuA/6FJOPSfLXlqtRSzi/vd51KIMvbj9ke3aE+2Xu2Yp9V0Y2erRvAA55yNsJeuiBr/8/7cn7gu6QoNd9Zp9SNgL9Od5OE/J+j8BQq1OOD+HG8V5VQKqNEGwZnf3KEx+lhgacHFBvakkjLYs/n7NgT/UQY3bGwe7tnZ7XP6RKGuFAaOYGNZeIsueEI0OdWwQIQza0z3Qm2TKBR7DY19X453xl/BVWsH1/TW7Enars0tqL6V39YwI1dQ8qyhUdw04HDZvdK19fUwV5ByhcI2YlVRDVZxwyC1qG7UKJh6ZeGDRzMmLEvBY3F7sYqrbg54W6+Mud9gCXVXeJtXrRmd3U7xMEomvh4i0mpVpk7j0xzzDH4i7NVRSc49i3Jdc/+ZdpWqD9SHMXIeEPCWzp3aXmBdqU2+CI8YJb3GEAS7XuaRHaLbf3CTcS8QejYs+0j71RULYCCUF4Dc12YjkP427bb0i3OwhFBBl74ai299gDwv0idSRA62NvkhkF8hLwbrNeJ/KOGJc9OPwDq3QcAqxXpXBURBiauGANfyE8sy1MS7O4R4W5ktp2Qm0hd7L0Qi8kOPIPithYi8NDLUqoGhZHcmQJvIqoPOf8pm2a3lR6lO7ZDyLUrQ1/Vo5aei7qMaMxu0F2CGSYTMPB7yGK+Ni75YXYPtwQwHmpEgUgJshvLwYuxurhYb+fJdPx5MQyDWn2n39a/ma2HmeKJqhfBW+9U+9C0KkqHh4Dpd6t874IEmYDFIBsJPQ5/s8gnVE+3N/d96uHuRbBHeeHA4qx9N1hxyAHec9DHBO4Nb/bbD+4QXwKOf2D/w8vI1GaN5l5Knqk2yDf6fqB8Nch41dAq127oCtxRHSQjVcwXA/LvO63uuAYM86Jzlrtgiuch+MqsJ5K5qlosF7dXgB2HWnd/m/Xs1BCdFouE8Eau1cFbDiNJu/xiEVfEJu7JXZ7QKN/X/VNNT+FQKrJA84j7vWk3X3hvvlOUahOS+5ZufDDit1c28YoLD7Neixsqk5GRd5jwTeDYEbPAMCPBNnQViVqhgurFYZwG2dg5V3iblRMe5/KHZW6QKvt5/eDt0SdhcFvF7jhe0mjiKT/cegpwUmdoPAwiY0CC2fHcKqdCXgQPXZEUHBh1SrKUJiXkA3q0bY2HUMfkrsDOF6xOzSuMCdCLx/B67kvRSQwnQ3hFQw3qHnld5xW3i5Mdx+VWtICHr8h0PR4+GHQITd2WErS8pZwY3jBbgp7NBYvioaYYEx4CZ1WjMD2uxioQm7axouBtiwXgBzcQCXwnCZoBb0OmHojyMj4ItgOwP+eSYeDyBf4i1USAW3Ly5QetcnheUGS8wwlQQvxLPCg0vuyhd2/w16jYbkkyP732kYIGwvYGaXCqDv/44QTsHJyqlk3LN7pfi1kDb/HEpWXh/soFDdntR2tzzr/wc5WGFfCOvi0naqAOwVvrAvc9aD+RQl0sF6AU5XCY0HMzpbq1d8qK1dAZcErlOisDst/ENPmbq4uBtrIhp7KUrziNljIZQCpRggpTDumYyMQKzBc6vt4gBjUIR90hIugu+Dt/4Lrf9OXNqOm4DF4Rm7rZRqmCyfGkIpZHmQ31FdpS8zjzWkyw9egD1KuAhOj8Tt14JL23HSZwwJblk+r9Ti+12bKyCMgpeVGym8+ztCEXbvK5Q5zIldEzjcbtTrVDSZwx6+pVPUhg/Hpe1cJkwTxbnQbIjWcekfOqSGDk9Udd4nxzSHIOx6OY9vcEUBylkicW3Z49HidDSa8ylwYatkuS8ehwJAVTLevADmAQihWLwApp/zcGjpsFbuvsTrVuyWWBQAssiLGKR2nPOnpmY7XRRrEvLkxLgVgLUjEvbXN/ywJDC6p9x9NUDpbbJVasIjifOC/7k8L/Ng6VwQzXqgBbDWZbdVNHugxc4lY6p1e2G8Ak9pTHKS+jS43gOlDwaq2UUUEgAPwm6JiELYKTa9oYZQNTtTRArASriwNS68fKfILL7rlEv/NoPwiUoBWFshL2b+soXdMgqa/g7YLdG5tx5s6BrwUdbCroELW0UB+A8U4z9CYt9hQFpsMtKuUL0OmRotMFO7M3ZLZKTmSj3Xb+Btf8fU9FfBha1i8e4S2T5YTtH9PyLcufCgY2M5X1PhTPyj7E2+oMKXOgy7JSLUywGpzSmX0UDwN7oCsG1kSvHDyjkA50P4xKQAUA6L6z3Qks9a7pdiLdS9e+sy7JaYBJ3d00Mv+L2jWQ+Oy+HC1qgC2CCeM5Ev4SEx9kd0AIxKTh7rfA9wyXHJwm5ChY39EnZLJHAcMjMDPQi7D0jbnSCKNeFyRVzYGgrAshFZ//d6yos4EUIoFs+g42FRXBodRBVIYj5z/MV+aXoj/xt2TARkdmMkgXWoFP2OS9vx02pni8PVS0PM/DXH+rL8WnDQJZmQmC8cv/u2QC4BstjdVwKsj10ThaXzrEdhF48XIDGDcWk7ffe/R5P1npp7POdG7AdBFLo80MgLMo+HsuGvVvhy12DXhC7o7JIoBetUKX4cl7ZTeVCD9d/p2rzf9HT+FwikoOXBhdVtjc4JWhpjXxEGCH1T31yCsIvDC5CZs3FpO33vd8L6H2t+xNYQSIHCFUyZ+aq6VTE8pAONPxqLNjtzaXHuGPYFC2Rc3C6F3TERnInFylsf8zbaAofqFbK76/RHyf8exhdk9xN3pnJfDpii21WwFu7pJVqD7wfvBeCENVzcLpW+VSM4Ew+Uu052dQimwOC7kcNXGp7QoBQ+1yUOcG2FC4dmNFxa3UsS3T3oNeLDyYcUl7ebWRChhwNDGADFjYdAYEqh3VXpffdtDDdHYgZJrwEQDuyCL/9SCN8LoNkeubEUgCfCD/mYRwOZmDgfBFQgtOQT074YpvSeNw3ry3Lzk8QYpfgfWgOHZNmmpjWQuPCegWv/G+PydvKeDwhb0bPLh7NWVJEFQjEAzlRSiIdL+/UAv3A/tS8cSxOQ6ru0App0Zz4K8yC0w9m/GrkxDdX9j6bphTwKugj1PB3Qev1CHtNpIKjK9ghxYrxSknRqbg31cthecWM/iizXIFyd9wVmHR4StsKkPBO++s8zQb9fjQlvPU8GPA6CqkSkPb55V/H9rhHmF+dEndR807Au38q7tPLZ1cI8PWmF2mYnDdhjsgIu8R4Ju+2Ctv4T82qAOROfIG+qVKX/YkVDeDDtu14hxz3OVfzyP8mkNVDWuz0z0EvihMBdxO/iIq/rvH8RdIgntRsFu3Y1uwMEVil7YrOG9nhKrD4xIxSFQkqLMAl2mmdYEJdd+je2rpGJnSpgobAzLvSKTborBj6Fq9il5i2ETD2T2AUlX03vnX4Tx92XmBvV24IG7Qap5ObeMfBSsbOCXTt2x6ZmKC71br3PH4Keda+Z7+TOM7YCBJev/UB7VX0AmDktlsWYVz9WbE7BrvOq1L0auLv4J9KOZwzXCxBC74SoFIBTg32X3H8iMe9FEEK5HYLLA0VjtJeV3+V3QXs5x1QCPGQ/p/lO2H1eLq8l4nAZm0sDFhLj0yFOcLF3Sdh9TiWUkwds/e8TyTqOkJkdQPtc61dGpfbouBZmoJ2FFuZn9Q2e2g2xC7UVAI1xz0oz41vsXAFfHOvigo982BN3d8vMp/CkgPY8kFs8vMOPm96x/xej5XiiBy335ygGhcQKW2LsXo/Hegzb7Znld+CSH6uweynoAWCZPT46bwpKArUuf089PuyWMcdGhngQGj9S2ctK2JUKJHZfpXf2m2IJ1NIBW5DTh1tNEcDQn1a7QLDvbpCdtj05UWHPjviVLusPldzH20KQOaSY8HeTp7Dmk3FXc2T2n56Ex3C6rFbB7nSq5XINe7OSm7evave4kA9NYv+Fy75DS+fAsGWZYoOXzFxI3/8gNa8KcEPR5e8ub4nNrXbuCliR2mWBI4UDWOEAri6q5ZQE0gdSz8qxLT0vwPqBXyZ9cOGPNso25NLeNjunWm93FvTsGRqcT6bmYWizi0Gg9XgPTErvp7/HNuf7VWPhWuyUMrjFjyfgN7q4tsZudXJJXae0sY9u9w7trbgPWoKOJRe9AV7B5d/e3rQ5nyLss6CYu5Ga/6p7GVJzFQRaD+CBVKl522M4rH+1et0kdm3PQ2IOQyesHsDWiEryHyloPDpaFEOqp9bMEUntLoGfiRnEG9LYlv83wc+wT+yyqt9/ZOUntf9QC5HymQZ1XP75oiQLh3mdb8JVdBW0KM/2bFlcJnWaoPvw4CWdzX3LaJr1Dorvf5gkooYMJ701alIgj64NvYKnGPjzguI6HNlByPRRpRyLvSHYuq38/Uu1vW/H4cv1qrmYfBln5kXPFkY/CUGA7ro831Da3CuNJmB7qzbISexREShbS0qnr8bK+P89+DyNQkHdVHVqX0cKKl8AOr+vGV7RLit+vaX1rn+P2H+rH0vx3UgjMYNI+10Yu7qLcMKQzuZu7VAAqWbF06Cg/4Ucwj4XS9Nn/bpxLP8IGngVIao2xaqHvTu5fHqp9ZVP7fIQcOOgKPd8vKRE2N7VX2CZka5YB955r/htsLu7YpGaS5SEzwGdullVe2mbKyI5Fwv7jTWWcfnn39I+WC2SMNgBinty4FjDkzz2Vef3XgcBN1ZFfGV/CeujTbltqBwN7uNfjuuxT9AzxstmSD6RJCZplGiOLdObQwN6h8tE4wHi3u2Jeaeibv8hQTf6GcX6l8olRY+M/fc4f79GK3U2hJAMOCbc3Ic716oPseukW2PILcz1LB6a6FeOIGoOPvO4PCt0SyX31vVd2A/3q+aCxBL/5Liwr94Z/s7cY1FNM0vNeYpr8XqXSrwyc42S8rEXBN0oSvec6tP8xladkdmlGnPhxfWr2hFu7JowHwQkxYwudB5TWvMVu3AQ51fVwGMaHMX7MrU7e89AVmn5bA+PKrbJ3dc0Q5RdDYFwcqiOMvwWBN0fZ2wPteZLXUmE5QFhDU1RGfBAiQlJj0g9Nvif+9mqxLe6qmil5krFA1eTxK6YqOV/F+9FnArAG1JDHRua4125xK9bCrlSNU5iF29oWdeSzyp99su7d9jQ2QKXDsNx58Q8VaKVQnXYdruG9wZwyZyOxXNI112vdkbd6YOB95vv1FNmt1dtney6uU1q95e4anwhsDVUhX5qF+rm59lNSeZd3KBWf29psVua1S/7wGJA0+jwvOPUPFuy4HpIrOCGDceYTMUF3Gan6WYY4hTVyymmWPSoVsvEMo421J4BMprbnBvt+ha1382KhsY1dclFlfdNCY5seDUSRZXNqyWfESuhPdCJgCtbCeC6cdYQG6Iec2RXM9ehq6znnXWEIialg/KF4juO2/rhDPHUHktr9FkgFz9dUOYs8d7EjMSDFXOOuAdKfSGJy0qpRKgKXPWQmvNpHUeUfE5G0B7YChf92OAM6DJjMyNn6jZSnEyr9r/e6YyJ3Vf1ICZ2wUqEztiVyOOPyzojfGmywhY7XKKqq3SeWr9yTnkUWpUZlfdqUmiXOy6Wf5/8Sp9lY1zwXbMAeVLaPQFYNpQQZy6nf5664p6XCVT60XNTm3o9KUVyaKYYCuhXqZwPHh6S2YOLcaVqY2uNzJZP7THVmFM+StjpQkU58pmMv+7Z53td5X3W65UIHRngVFZpX0ceMrs6LvbuII0ZaIRlGKVM1BDE7hNlUlOXhAtppjrrdnoPD/EGugeTWhBX0ptDl01q15FcCqlyMR/Wf3GRNy41Z9Lft1G0sf1xwc2JNN3Daf6fHn/Gmt1dyUN3RMWMmVlp398cUBnsR2hD3xMXTmpOCijJKZEBJlWrFkjM7UoX7Dw9d+GZJ1Q70zVKV0jOr+GwB4dk2C3KeS48Mputea7T57a3PJmR93fNLtIw3eIKGaNYZmlanBgOrNRl5kcVmVYFecbKaWbOkTkT4Vz+LaKQgB5v/p3V3Jr1HZqnKpMfUCTc/ayw+V90dLAX1G3PSVn1oHFRnfYn+8tdoxfu46/zGePtRMdVEuzF4BkTYTXAerASuTHhKAE0Nzy8uel3kHLyj8gF4A7Buj3/FHyX65au5bPjgDUgnHCcmfdV2x+7hLtp6nzOC6N7d+y5y+xBwVTCjCr7zmi4KjJP1uqcMlI2rNpnU/TstnNEuaYaYy/ZDTckn9zZZ+TxnJq176m5F4erEY0KxfAi5xS4HnxUhMTaFD7r52OdTBjaxc9NpoJsikUhmsRujYOlGw7gWNjdAb7830mgXC2tW2OhuFhHKAiU2xXe+2HRuGpBBJ4vOqea8WIuq9VAq1tnveW6/oy/SQtXf4AWfyH/s2gmXUZPUd95eCmjG7ui+Wfm2ihCAzwMSWMNOJHMvQCYkNZ2sOp8dv4doDHQ7PdfjNPWKR3WmteRmZuCNVJSc7LKiHKXoWDE+8s4xHaVYPujFz0E7qJ40BIBu0CfVfjeX6i5E1O7mW7sjrrrgeojlRCq3qSDdD+/QtiO3decUBeObJ9Hcn9UEpSdTpTdDRNly9YQ9UbYums6w0KnKzPA/bnUdCwJLdfnH54f87RqQmCsuRygaxSDxwapuoK1J06mdhslz125bWqL8byriXdGxcvhVFYMoPWaDwcqjJBAL5k4pznD210fgd2DqD3nqXg6tf/LKQu/hVRDPzwQChp9la3/46PPJSmqF75XkE/3lfJOioz+3XQHMTn17J7dcIOUoqDNLhZclUDHVsKXUipSZvlZYl5QOByDvVyeanML/hDi6NldRYrEP0WXsnnAnyJDeUYa/epdVu+MC24nzc17wivv7ky+vUcG3Eo4SCEj2qR0hLKRaJP3k9W8ttfwAPf/Dm3gSXfgaXiaQiM1Q6VzHqiY9U/NWTSHvbTYuTx6MlZXyoPRHVXL4RHO5VHtvqgiqy/r8TwH4FPbJ02NM7vj2WDv0QY7mpSBGfQtaK6jVfgOPstgUrunspfmHByiSrn+N1bO8/mv1+/DjWZ42JZ7OfSIknd2/nbD7POoLv7M1GD1x0oRKzsnyHLBsZURcmMaHoSjNXwoNc8pHJR3vQvA1LylPLt7ERyiCiDtYs0HintlWCkeo8ycpdLPhD1sLuBwggwxCmQqX3fXITOnIdZfBVK7JF0Wb0e4CT8VBYYT31zRbKfTyf4n74VvOOFQ16p7BS09q3D+Kd9Gc5+UlT3POU+hhQEKF/9GJLfuDGswT7fk7tPisQAVgq1prs/NzA9Rbkq2drnSoafzu7lHv8bn4zbNpbh2tQak/JEQuBcOT8RwWEp3kNgzpVWNFGWxWelhAM5fYjd5ai6VPiBRXvrtBldqt0UVUJXhC1Rr/K2/xMH+UjbTnE9Rx4Wp0DPBvFna+xSPhuIkMJ5BkNoZcXAiRMqDNcJdI4frHHrn6lKA7QkK5/m3cY6D5kuyZpem9T1PJRfB78X/m5T2NcoIbNDUnkVL85rj3ri/i7bOLruuKAMck9PplXBkySGe/ZTX+S4cmCjP+K7K+6L8KXrSMU8lDLBNhwoVTyTkS191iqJXGXq/lCOChg0L7BNNHeq4hxE9Sd9n704tVq0OYmW5/0d+j+o5HugNEBVFh9CvVd3FoViMKnu/XenlJLiaXY9+x5X03z6pxqUvIdW3qS/EmjgooEms59ScG3wnwe65J1+V3vacyf6/mFZqbquU+39US2hZ5fUchoEfEZGaG5QVwu2C+a4qEwKpr0FC1nFmhldGJhbfa0h7nL8XDgkYFe4Ulppbq7XhZdNTCZS5QgZ+VCH7v3Ohf6Wy1XApDkkUrv81lPfBc0ElinEDosrJLOcK/OeSBI6Jn2DcFwlN8FOZuFXJZ96ALKGppM2y7vddEQckYLgTqOawnxAS/zpWft+CLOrQ+KHwrj0CnT1BPRfKKnTgn8chGstwo+AEod1FWaBkaA4StOv/v8rW/3lhfm97DGTSKOeU8j/scQjbgZ5RjKhcJ85uVuqH7JQw35di6VfxvU/DwQjS9b+wWOiaeSCh9oNnTxxkEr+jz2gfHI6+/cD9xcJDe+ARGMkaolBJiBR9x39TdQNzFzYQ0vnsLYmvqvvdbh7kd+cqmGIS6icNfPEPkxkn3P4dAF0LM18VOQIyj3wTiq1NH6ZFRFa6rsB5R9qeglCs/wOV3/ejwST+8cAwGW5kTqfP9VS03U1dzSCp2R1wFkEZB3FxqRqIadiQXpXBXdIQiGtr6+lQ6JoiGWywcinY8TgEQVz+c6hUuPx5+f/iddTvqDJmKvE8crVNau6RUdXwPuaF4mPXRdteEIoAurB6NbM9HG/MSgEnJ0kjEWpO5PuwsuDUbiEaYkZ4w3nkuAGWquv/WC/foyWftRikQ+19E3Of6gTDOGXKL/Sur8aUThAm0lDIHoqD2+kB/kI0d86kTvOdSNAtJaNaVZUAc6Py93qD4o7jY/OXhOwj5UoX17XjnJkujatonkdmLpKBQqpdC6P3MH4kHpAsnxobHoSPtKalhCE52DjAXfQWPCjzzXmuAQtHV21W2+w06q2e2csBynCPz6B+cdbsynV/vhY7JV3yK8hFz51GeW4HjIPuXPyP0/pvCgUbxCykFpQOckXSHA51t8dzmmfpuYou2cMk8YnHu3J8v3vvYEf1UADckmW4/u9Rrvm/epyfg/eijBymhFguP8vMNXTRv1B0nsMZrstLyIZAWTkXAKjAnagKl9/rOOhOBMWH7a7Ta6ThB/dmZ2uryDXo1cFloRwnpuEiyET2R5Zvod46lhPwitj89PTPy8kwLfb2sGKQmafb9yDOYs/X2rRPNd0MZwhUn1q+KAmQi0mQfAMBoGSRSwWA6U/PdbTOJ5GFfrQkEekqASdhc3u5/KdWt7DTEe/S3mlT3zONffEPEoWqzc6MTQ0aj2LU5lYkaB5DKWElBBr3iV8SG1tbATC3YL9F3JufQ6LsUUEJHwDtsOua49yJGQAhEbWAa/vDdQx6Dq8jV4qwlcitfiXcg30WmWL8M52LOyWHBy5+AMYBJw5yly+eXQ0BEqsi8GvhpjYDpVSQwxGpuVfm1LMFlJozpb48tQdIbkhitxYByY2U2DriMFFq/yG14VzJwDkkXF0S+mXNGdtc5tZsp5P+GNyWmVtGczZ9Zv8p1TGSlGn3lilt0rFRSuH60s+7aV36FXkytG7F+v2KvRRpGC4z97fnTaAvPwB1CVQuieMSInQAw/NnwtTPTUn+rQw+4US0ooyyRnukVVoU8wWamlfo54vtg5Geae+/8GTRwlpCTo8WiVfmYfnJ/87/PTNPtP+5/lKBIbMvzEvSZ19Gzpp3ZSIi/87id3/W/lnoMxmLd9TgSm9qHpJS3hC6gQJQIWWg1x/KgAh8CBw8ePCUfun/IO599mBh9C4AnjwDXHee5icWFh8EER48eDx25svM5RLScd0lEQDQTThOzDHVwq2LsiU8ePBoXf73I8EVgFDhjGmeUAhhhQcPHvcKwBAIWQBCBjMI8ODBozcfYSYIWQBChGNyCAPgwYNH7bFbQtACEKb1vyIEFB48eBTDABdD0AIQIok9CgIKDx7E6um5mTyCx6sMvgIABAg34nB94FvNOaL1Fx3afodwxYMnuFr8p6ks+AyKz28qbcX/hzQTM8OcN6PiZGMAQEBwsyAetuH2sFsaWjT5H7+D559LQyK7n7S1TUyCDnB48Hjtq/+yKOTcUrnVLkDnvvc4jAL3VUFpvioELgBhxf/nLcXdx33qM7uCKAU8flcGGnEvcAhsPHjqv+ypxTK3Y07N+WTZ7yAzQuqZByFzF5x/viMhcAEIyv1vd1FwL15Y12fhqV/crZAFVzHLgPvLfwzBjgdPB2OkE5PS+bhLph2mdiOy7Gdz1nAntQspfOb7IHABCCv+f6WCgNrC6WdssVOSUrASCbq9xI1ZTH/7CBcBngaI1f/eftHfTc+pMimPRxxrt9OVOSLmO8cKwDAIXABCQmMuAM9h9wHnGXBuQWa3F+GY5Xe0hxJ+xMWBJzKL/nOZxJiZa4uRx/kmMhaZvWKlhQdpyqPz72lngNAFIAQ4K5ddiW4tlg9K/17FSOQZxGvAY0Yzczp5DW6T8bQiaHHh4PF+wVs5G9JxUy7542h/bkWJcUs0Dc4nC9Q7eJL7/CAKVQAAAkDc6s4TAG8N/ntzAiInR9Xs+pLslJozRUHgTOliXr3BpYWnm4rvbzJyu0jAu4H+/RTaW7s11fI1KTY/d5QT8DK7rkIlwIkQvACEccAPVtDw949+XThrmnuXF6WLm8t34nppFuycf8BljK5LJ/GE+6Tmm/bS1afa98B/aW/sK/XzmV2KFMrpJWZeOfmQT60yGRAAEMIBNzcpDP1YpmHWj+OznO+Q2iXFmyDhBns4res5dEk8jsszSkv+fXqX/xLlj7Pqh+QTNbSMYK+GW2VqKAQvAGEc7sS5G7TRBeaoCtZpuFAja5hTs/Nh446EhMYcrzNX9QAASrVeJ3bfjY/a/oI/eSMfv2iFrD1q1bzWNFDitZuQ5bp1uyeCchvsITLngeOuRSIk91a4hP75CklES8yN7bkPd4trlltCJ+bR9v4L/SWenZjn6XmhPUP9JcmTSM0r7Z3lXpL/zn8mNc9Jghu7yTPyfiTmEfr5IP33e6U6Q7xN1PCJy05Tcyk958lnyuwJ0iAmswfR591TekBwGWmRn7G6eJQ4X0P+Xm1XPzWlAqMpAPYwdAQEoHoHe3kF92kfLOxocAJYZoZ7GLd6fHU9KaQc6Lv+n6hkHL/HXkK7FhQtAKonVPdSuIR2w8J2KER39FJmlti1K7d2Nbu0fotoSugceRgO+JNBdlqF9b4CCwtAqQoAWevum3wsjoXtAO5LwGNW9a3YLykUMEtl1q05n6IYUautPFHSHxibrHDcdZNCRwCAUg/1S84TAMvsWhY6bXZS5xnVHWdZv1KJ98BKU5E/oK00XYvNOQ5S87DzgUWuZhYAALotXHsV88A9TwBsdDihjXu76ysBl8Z/6dgDPIRNBtPvmQQbc1zvgnpguC8XngkLC0ApF1H+dwWB2hcL26WL7VAvJW2cTR8rMt9BW1Giv7+Relb0yFvIszac5wutjoUFoBwBu4HCgT4YC9tF74uUyHmoaW+zi0W4N6eSZjH6cf+jsBm7/E4WV5AXe2NhASjHCnVf28s9z0HXaLPT+BlnTAl0MTVd8aUcZeZp+l29sRG7yBv53xR6hlyAhQWgDLghi2uhyuVCoDtW1Sp+hg5RU55YLjuekKe/Hl8h/lzPfqV8Cbd5Kg9jUQEo5zC/6nyWOajHE3Osp2E2Z0awFuu4tzIxjtahzHDsmTFtWFQAfMPlN4n5zvEF8ywWtq530UssdC+97u2W4V7++ex+piuaC7Hp6vYaXuC8bBhhGAA8o9PZCy2A68VfPsCP5PpeJLjvz8OjsvwND16Qt2itJ8SGq1cBoKQ950OB8lmxsAB4tbYUZgCkdn8sbA+o2ZXIKzPCy6hbVjhC8kZp5KOM+b2HU87FPNhoPdmjlOTrXm6shoUFwO9ls4NCSdXaWNgeK2aHecoHeDaYToEaVmXH4Y/tsMF6uj85TON8KuBOWFgAvApdc4rCQZ4dC+vAGvbS+jaQkE2Wr+ipK+JV2FwOeDr/i3MvFY+pBgB4VQBucXyZ/IpkHkcMziej9RzoySreq7Tv2WZnpu/5qQdFp7lpqP0rNpYjnM+yMNdgUQHwqwC86LgEMMWiunw/dmFa0588dAoka86u4f37cdJfYl7zEvfP8nmxoZzKjv6O39ETWFQA/Grxwxy7WB/CojpXArb3FAr4uqnVzu3texVhjhv85DrYbbCRnCsA1ziWHa1YVAB8wclf7lt6XoyFVRG2F3tSAmpNzfkUnhSbgxomx6GKcMzerQLwDRYVAF9oTAHk6XZAR1lzHa7pXBD3o37v4ytf/ut6an38poQZgIYC8B/n74vnDAAAPMD9590L3S2wsFoC185IF9onnpSAq8VFr3L5UyzedffJDvMa8m9FyQVKBoRdXyEZdQ4sLAB+FICtFebOL42FVbWcV5C2qX7i5oe592TQNEJflQ085hpoehAXVXhny2FhAfBjUR6icIBnwMKqv7c9/MTORQnYzNnn5rBCYp7y5ME4GRtFGY024hjOBICvi8Sc67yUDD0AfL27K/1Y0eZnJ1ZZkfF/lafP/Cj2oQd4jZ0nEdvdsLAA+LlEbnUseIdhUT3Bg2wy85KnC/UL+n1z9tBrcainjP8hpLBMhQ3iCd4bbhWAI7CoAPg5vE85vihew6J6pCWf3nkfh7HNa6/3Ys3yTdxbih3uv58kLg18GhGtjkM3Z2JRAfABt0Z1e0k8gEX1LYDtkuKm96MEvNjtVrqcFOqjkyGa/ZQkQ/JnMKsBgDgVgI+dl46BMpSAbfwlBdKAoq7G17mkKzGfefpc52EjlOJFvM+xknknFhUAbYrZ6785FsL/xcKWpQTkZ3hUAi4dZ48A7iaYmMSTZ6K/TKcDZXgA+jreW09iUQHQps1OqtDE42AsbGkKXW+6CO/3WB54dKefhTvvpeZZTwmK7yHpr1Qv4gWOFYBXsKgAaDPQzqLQA2BHLGyJtOQT04U4wJsSkNldO1BEetFnuN2T5T+cFJGF8OLL9DyZkzAQCIDYaLULKFwIG2NhS1cCZvUWd+de/qM3buFYfIxNikCdCgB1i3Sr1L2PRQVA/+Au714g56tiYQMgscvS5fyLt0ZBNbtS+5461KP34QS86ADI7N7Oe04AANQP7rru5wCgBjug97ult8tYhu7YY/z9PgoxaA0qAt00JPKdHHsAfsCiAqDvAdjMuWDuabc44Pod+7uUU1+Wf/4GRsaGpGjS9E+3CsCvWFQA9C3E7TAIqOJIqafjMq1SH+pbUbMz4cUGpQBs4vw9w7sDgDI1u7vzgzsknxwLGxgt+QRSJx//5f8jeRmWwAsNDB657Ppdo6cDAMqkdn+FEMCEWNgAYcXMdc92n08xR2ATvMgQPQAKuUTcRwIAoKoAHAbXXQPRamcjK/qTKBUAri4AYVLL13T+vrs7bwIA0E0Se5RjK20EFjX4d764NM+Jy/V/BRTLoPfUWs7fOZI8AVD3ABzrvB4chI+4bM3vkbj+H0U8OHg5sg48AADEdxGc4LwtK4jFatsxgsv/HbpcJsHLikGhRA4AAJFp7o57eEMBiM1yOzrgy/9DlPtFo0xuoJBL1BsLC0BMHoDE/IRFjQiOq6fmkvAuf+oqyHMqQCyK5ObO50sAANQP7jGOD+4vWNTolIDe/ib3dbELXGJXwYuJypDYDoYEAPEd3COc12ojWzs+uHdDavqFoQDQ/AIQmxzZzbES+DUWFQB9D8ChCuU742Nho9wLk0iP/XJr/ffHi4hSATjQsQLwERYVAG0Su6+CEEfWdryegGnIi5OW5Po/DS8gWgXgeMd7oYZFBUDf6tvFuSBvttNhYaMW5gtLHN73gB8ojhHvGXOO4/3wJhYVAG1qdivnwrzFzoWFjZSWfHoSvs0leQBewiCpWA0Jc7XjvdAfiwqAvgdgQ4WZ8JjWFiPFnICB5eYAmLclDAFi8wDc73gf3INFBUBdAchXVcjiXh0LGxmJnYeE7tBAGgClTQPtLHgpUSkALzlWAK7CogKgTc0uoiDEt8DCRqUELkGX7ueBDf/5QJQSEIkCaQY59iKegUUFQBu2tNx7APbGwsZiuZG3JjM/BNoK+HOZXAjChvt+8BAwtzLkYCwsANq05BMrKAAnYGFjuPzJU+M927/bnoDvEVIK3fq3UynIkO2wsABEqb3T7HYQNtxwh7s2xjAOODO/oTtgwKiEEaH0AeDJEjRDHCfwPIRFDVbh60Xv+6w4Lv7RWkyn9gC8wBDlh93Y+fuu2fmwsAD4UQBediysB2BRA2So/SspZ7dFd/mPqlyeK0oMCEkBONh9LxEKTQIAPJCaex0rAN9hIFBgcG2961Kt8kICd4kyA8IgMZdhEBAA8XoALlIQ1FNjYQOB3amJGVyNy/+PS+LlpkF2WrzcIAyIfmgDDEC0CoCCC69ml8HCBvFu1xWPTKUu/z8uiiGUgb4gXnLpBsQnjt/rnVhUAHyR2H8ptAPeCQtbIhyC4aS5xJhqXv6jlAmuixdemuxQKAE0p2NhAfB3iBdXOMRnY2FLos1OSOt/bWnZ+v5/pxFlB3kn/kntagrjxHfGwgLgC41mQIl5FAtbikCekWKyr5TWvjez29PP4SX9/mtE+QE+99th7mWHXRYLC4BXL4D50LEw/gQWmXdhvDyt+8clXb5tpEjO+sfnSPJvS0sOTOwM2AyeyPI7nL/DwflkWFgAvB5k84RCIuBMWFgPSDdHuy+9w99LcvsPaGq2043ymdrsYvTfvyhJCfgIVqQ3ufGB4z4PQ7GoAPg/yBcoCONNsLDKvGP/jy7aG0tszPNKU3M+RSfW4bzuPUvdah+8F7xQirDHx/17exALC4BvOGsfiYBx0Wbnpwu4tcQM/CfG2bFNLgmTlVgp0BdNg7Rkht1WQaE8CQsLgG80BnqwdQiUPDaSbPdjiZb/bXS5T9BFRYW6EOZvlDhHYACFBObBpnHuNbxWIQFwAywsAL55Ix/f/WhYikm32UmxuA5hizs115fcj/8Scq337qa3YlL3HeO6tRd/ICV3K2wgRxRTRIc5f0+j55IAAHy59BTKx1K7ERbWEdKvwdTKncpnj6o7rs4leqm5teTGQX0QEnCyFxdU8NQMwsICUJ4CcJ6CtXgpFrbH1lavot6aE9tKuzh/J6G/o5vvYs4vWQl4Fy2Eeyor7DEKsuJ6LCwA5R3qzRW0+g+Rid0DBtpZaA2fKr3dbmLXdqjQcJviQ8r1ZJifKY9ib+zNOsnM6+gACEC13Hoz6AhcuxQWt45LkhP9yh7kI2V8dmE1hTMxv5Ss3DxIeRXTY8N1gzY7p8q7qOV/x+ICUKoSYBKUA5YMX0ipuSeAQTuvq3fVK7oXflny9/xSBmKBLlr/9jgFRXMwFhaA8l17F6l0Zutu1nijWv1SW22+CuDyv5sqQ/7mR+Gxc5We3FjEoG/otKkR+N8e7UXvaqBKciYAoGQ4a18nDICRrWOj1c5GVtAjQYzYTc3JIui9ej3slHQJ9A9A8aHZ9nZjbMhOrf81dPac3RCLC0DZFK1lf1G4VO7F4nbA0/lfSKgeWN4EvVHcsD+VWivPjYVSc1UQSlBmbmkaZKfFBh3DQ3inSkImyx0AQAheAPOQysz2zM6BxR2Jml2a1vqtIC48TvZL8yVKX5MiDLJ/sV9KVwK+ksx0VAq071dK0tN4L4m5D4sLQDAKQP4fJYF6ERaXrah8alqLK0jw2UBc/s8GZ+1y2WFqvgnEG/C0DDaCYXCJUnhwewgFAEKBL4PEjFBxMTdyyVXh7t+bLpSvw7jY5HK7QNpAh0iRHNgSyDr9RhfgmeMcflRV2uzM7luFi0z4Be3CAQiNxDwKL4BL68muE85lJu/hR7Kyt45g3SaRioRQ1o3736d2m4YLC7DHSmdN74CwBSC4A8/T5pQsqUaazCZTFs1j4Vz88g4yqjpYIJo1lNKz/Mgw8gJGmnSZ2OUaZg9rrT2m/wEQIOzq5BawOsLz4cpbUNwtLTE3BhPn/9OCvT1al2stX7P8pkEdrGernbuy+7iY3fCckiL6sYTFAABBuv366FlQ5EatIty7X9ylNDwnLKuf4rd2n+gVr5Z8VvouLwe2tjQoyVym3jWxHE/gXorrdiqELADhWlyLKh7+r0WYV4U0n7394v8trMtJ1rqNrP7FKuSdmkBlcqWLenb+XFVJdE3tP9T6U3BIgZVlAEDAJOZ5xYvpRRHmcVtIC9P3uEmlasLN01cS6SqpoNpNwykVHK3ahUcec+Z8rBQNwQYorhOS/wCI4ILbWNk6vS46tzTHRWt2PfrsTwR66ecyRTCGLP+ewi2UM/NSmO+BvUHmWlIE5o9uf2fmLtW1qdllIFwBiEMYKA9qoWmBMSgBPCimaNtbC/biL5Isn5OLsVHgPgaZOS24hMtR9/hjMg/D94yF7p/38SSfQXct+kOwAhCPF2B7DwLy4iAnBrLATuwqMimOY7whX/xFot/hDTt5sXhPQwN/RwMpJHMYeQWmCXCv96bPd7m+gpqvCqEKQDxegN50+aUe3NaPkBCfKojvzG7bzJxCz5CwL5Q/rP63JR+h0WEvTWpuDf+dSXjgbpm+GUInRvFumQc9fG9Y/wDE5wXIt/AkGD8mobiZ95CAuD7t4jIKN6iOfV0oQUvzE4Nt51sWPNUwqJbLY32HX9K+u5L231qlvMeiv4IfRZcHYAEAovMCjKfXEKSTCgG2jjTd2Vy3XVwU19LzUTyX/khWP5dqgo5J7YzSdCqud/pN4cGw26n3FWCFlyfx+RyxDACIFK4l955oxRczDazh6XA9mRk+JJ+cLvuV2kfNcoe+QdFd+CMPUMnsEbD6u6i41uwOQZYLdu1dJ9JfIs13krBUTzvncb8KPgPemynR7Ikq9f0AoCHhZL3yhKEhQd5Kn+FOes6iS/BgEma7SLlbZv9N/7wtCfvd6eehxf8u9fnPS1gh1st+zDV4qtKtZ3W9AfdW4P3/Qt/jLXpuk056qd2T/vsmRQJkvgT9+0JF7367rAygSu3O9GdPKsr6Ssxn4cRHAED0gpSns31QmQs1HsH/mViyjTaFzrU3gPNLMvMJ9pRX6/9N9PwHoCqwOx6CzafX4xLJ0gZu4HBQai4Nu29AZS7/X0leLIhNB0ClPAHmXAg39cv/+Ur18A8Nzkhndzr2muJDjbMAABWDe/hn5nUIOJVM8KGS1wB3v4+wQO9i4l0sJYNRWf8PYg8DUF0lYFaJTUPYuRKYP1CM+ljK7v8bNpdnsnzqYqQvhVywF114rwYhbAVA5UMBdvmiBS2EXo/nyTfb6bChSqbVLkDv43HsyZ5c/vm3FF6ZD5sJgMZQAjaH5VS3u/+2phY7FzZRQBTVAuvQnn4He7SeFsd2dWwiABrKhWp3hfDr1sV/L3r3B68I9JJBWIl5D3u2ixUr3DIcANCInoD8PyitGqeQvE9asIJ4kIRXSRT8CHt4bOWqdhtsFgAaGenKRzFtCMWRheMIWpObpUsbiJch+USkCOxD7/ND7Osxav3/hQ0CAGiSed+Z+QqC0Qyn50K6NObApqgQbXZCeqe70bttwx6n8smaXRmbAgAwspCckyylAQ0qFD+Q3ucog6o23EOgaC38YoPmsbQigRUAMBaXKVvADeHmt1I+xiOM0fe88ZDhOzzGl7PgG0LJva5H0zkBAA0jHNeKevzu2C/+D2UqG9z8gGnJp5dmTpl5v7JDqdjrAQAAXWao/StZSCeTAPm5IrH9m2QoEruBARidIjywDu2XO2SEbxU8XKm5kvb8VHi5AID6aLMz0+V5eXSu0sT8VMxUt/+G6xN0C54+yEmDqekXZcOsxDzSVMsXxYsEALihZmeiaoEz6FL9MmBL/1N6rqV/3gT9+YETBtlpyTOwh1yqIbfQFkXF3E3ndBm8NACAlkdgQukeJg1yynaV0jCezDxG1trh0qyHO8EBoEVqJ6G9tjHtuT609wcHcvG/R56Kk2gmwmx4QQAAf7TkE5PFsamECBKTqjfn4TKm1NxQNHehCx/Z+6Dc/T+rtB5mhSA1bxcNpLxUr7zZnsi6FMb3AgDCgOvoM7sGPQeRgLpCYqjSfMV838U65W/oz9bo6S9lS5k9TjoVcjyTSxQBCBlJnLVLtjcdOof288OiGNedSMt5Nyajv+ce+juPp+ef6FUBAIjRWpqAwgfTUB7B7HSpz0MCbl5yW87dNNDOQs1JpoQ1DyoLW+mcjd9m55eum9x7IrPb0c9dJL+AZ3Kk+U6SsMqVCGm+BP2cESEtAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKDR+H/TADtRc4SxowAAAABJRU5ErkJggg==',
      }}
      style={styles.backgroundLogo}
    />`

Reply:
I think base 64 might seem scary to new comers 😓

Reply:
I agree. It could be less scary moving it down to the component file. Up to you tho. I was trying to avoid one more file.

```js
const Header = () => (
  <View style={styles.container}>
    <ImageBackground
      accessibilityRole={'image'}
      source={LogoImage}
      style={styles.backgroundLogo}
    />

    <Text style={styles.text}>Welcome to React Native</Text>
  </View>
)

const styles = StyleSheet.create({
  ...
})

const LogoImage = '';

export default Header
```

Reply:
@elucaswork Great Point